model/src/main/kotlin/config/ScopeExclude.kt

```diff
@@ -37,4 +37,9 @@ data class ScopeExclude(
          * A comment to further explain why the [reason] is applicable here.
          */
         val comment: String
-)
+) {
+    /**
+     * True if [ScopeExclude.name] matches [name].
+     */
+    fun matches(name: String) = this.name.matches(name)
```

Comment:
Nit: Could (and should) drop superfluous `this.` here.

Reply:
It's not superfluous because the function parameter hides the class property. But I renamed the parameter to `scopeName` to avoid this.